pkgs/development/python-modules/cheetah3/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchurl }:
+
+buildPythonPackage rec {
+  pname = "cheetah3";
+  version = "3.2.4";
+
+  src = fetchurl {
```

Comment:
fetchPypi

please read the Python section from the Nixpkgs manual

Reply:
I can't seem to get the package via fetchPypi. Not sure as to what is the reason for that. fetchPypi tries the domains `files.pythonhosted.org` and `pypi.io` and fails with both.